tests/scripts/thread-cert/border_router/nat64/test_multi_border_routers.py

```diff
@@ -164,8 +172,7 @@ def test(self):
         br2.nat64_set_enabled(True)
 
         self.simulator.go(10)
-        self.assertNotEqual(br2_infra_nat64_prefix, br2.get_br_favored_nat64_prefix())
-        br2_local_nat64_prefix = br2.get_br_nat64_prefix()
```

Comment:
I would suggest preserve this line, since it is 40 lines and half minutes from the last time we retrieve the value.

Reply:
BTW, we can make it a property (probably in another PR)

Reply:
The local nat64 prefix is usually stable for a BR. So we read it once and reuse it in all the TCs.